metrics/metricstest/metricstest.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2019 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metricstest
+
+import (
+	"testing"
+
+	"go.opencensus.io/stats/view"
+)
+
+// CheckStatsReported checks that there is a view registered with the given name for each string in names,
+// and that each view has at least one record.
+func CheckStatsReported(t *testing.T, names ...string) {
```

Comment:
For tests, it does save a couple of lines of code re-writing for each call to just error. 

In this case, it would be wonky to have this function's behavior differ from the rest of the helpers, so I'll leave it be.